src/dmd/dsymbolsem.d

```diff
@@ -5976,6 +5976,22 @@ Laftersemantic:
 void aliasSemantic(AliasDeclaration ds, Scope* sc)
 {
     //printf("AliasDeclaration::semantic() %s\n", ds.toChars());
+    if (ds.type && ds.type.ty == Ttraits)
+    {
+        if (auto t = typeSemantic(ds.type, ds.loc, sc))
+        {
+            ds.type = t;
+        }
+        else
+        {
+            TypeTraits tt = cast(TypeTraits) ds.type;
+            if (tt.sym)
+                ds.aliassym = tt.sym;
+            else
+                .error(ds.loc, "`%s` cannot be resolved to a valid type or symbol",
```

Comment:
I don't think this can happen. can  `__trait(getMember) ` return something that's not a type and that's not something that can aliased ?